packages/ui/src/kit/VSnackbar/VSnackbar.theme-default.scss

```diff
@@ -0,0 +1,89 @@
+.v-snackbar.theme-default {
+  display: flex;
+  flex-direction: column;
+  background-color: var(--endpass-ui-color-black-1);
+  color: var(--endpass-ui-color-white);
+  box-shadow: 0 3px 4px rgba(17, 22, 26, 0.2), 0 1px 5px rgba(17, 22, 26, 0.12), 0 6px 6px rgba(17, 22, 26, 0.14);
+  padding: 16px 16px;
+  border-radius: 4px;
+}
+
+.v-snackbar.theme-default .v-snackbar-left-side {
+  width: 16px;
+  height: 16px;
+  margin-right: 10px;
+  margin-bottom: auto;
+  margin-top: 2px;
+}
+
+.v-snackbar.theme-default.skin-warning .v-snackbar-left-side {
+  color: var(--endpass-ui-color-warning-secondary);
+}
+
+.v-snackbar.theme-default.skin-error .v-snackbar-left-side {
+  color: var(--endpass-ui-color-error);
+}
+
+.v-snackbar.theme-default.skin-success .v-snackbar-left-side {
+  color: var(--endpass-ui-color-success);
+}
+
+.v-snackbar.theme-default.skin-time .v-snackbar-left-side {
+  color: var(--endpass-ui-color-primary-3);
+}
+
+
+.v-snackbar.theme-default .v-snackbar-container {
+  display: flex;
+  align-items: center;
+}
+
+.v-snackbar.theme-default .v-snackbar-content {
+  margin-right: 23px;
+}
+
+.v-snackbar.theme-default .v-snackbar-right-side {
```

Comment:
Name it as `close-button`

Reply:
Ah, I see it below. So, I think you can just left close button with `position: absolute` and remove that wrapper, because it is just useless

Reply:
wrapper needed for make controls position without any margins
icon(margin) | content(margin-right) | controls | close-button(margin)
don't want to to margin to controls section